sirepo/package_data/static/js/flash.js

```diff
@@ -122,6 +122,7 @@ SIREPO.app.controller('VisualizationController', function (appState, flashServic
     var self = this;
     self.flashService = flashService;
     self.plotClass = 'col-md-6 col-xl-4';
+    self.gridEvolutionColumnsSet = false;
```

Comment:
this isn't in the same context as SIREPO.APP_SCHEMA.enum.GridEvolutionColumn. I think they should either both be on APP_SCHEMA or not.

Reply:
https://github.com/radiasoft/sirepo/pull/2676/commits/6867d3d5b0cd825d636f681fbe341e7967026ceb